kms/google/cloud/kms_v1/proto/service_pb2_grpc.py

```diff
@@ -169,39 +169,45 @@ def GetKeyRing(self, request, context):
         raise NotImplementedError("Method not implemented!")
 
     def GetCryptoKey(self, request, context):
-        """Returns metadata for a given [CryptoKey][google.cloud.kms.v1.CryptoKey], as well as its
-    [primary][google.cloud.kms.v1.CryptoKey.primary] [CryptoKeyVersion][google.cloud.kms.v1.CryptoKeyVersion].
+        """Returns metadata for a given [CryptoKey][google.cloud.kms.v1.CryptoKey], as
+    well as its [primary][google.cloud.kms.v1.CryptoKey.primary]
+    [CryptoKeyVersion][google.cloud.kms.v1.CryptoKeyVersion].
```

Comment:
What is up with the b0rked wrapping of the docstrings here?  OTOH, it is already that way on `master`.

Reply:
@crwilcox OK, looks like `black` is screwing this up:  the pre-blackened files are indented at two spaces, so black "fixes" the indentation, but doesn't handle the rest of the docstring.